File: sis/app/src/main/java/com/example/sis/datamodels/RoomSchedule.kt
package com.example.sis.datamodels

data class RoomSchedule(
    val room_id: Int,
    val hour_start: String,
    val hour_end: String,
    val day_of_week: String,
    val status: String
)

data class RoomScheduleId(
    val id: Int,
    val room_id: Int,
    val hour_start: String,
    val hour_end: String,
    val day_of_week: String,
    val status: String
)

File: backend/schemas/roomSchedule.py
from pydantic import BaseModel
from datetime import date

class RoomSchedule(BaseModel):
    room_id : int
    hour_start : str
    hour_end : str
    day_of_week : str
    status : str

class RoomScheduleOut(RoomSchedule):
    id : int
    class Config:
        orm_mode = True